src/wallet/wallet2.cpp

```diff
@@ -419,6 +420,20 @@ static void throw_on_rpc_response_error(const boost::optional<std::string> &stat
   THROW_WALLET_EXCEPTION_IF(*status != CORE_RPC_STATUS_OK, tools::error::wallet_generic_rpc_error, method, *status);
 }
 
+std::string strjoin(const std::vector<size_t> &V, const char *sep)
+{
+  std::stringstream ss;
+  bool first = true;
+  for (const auto &v: V)
+  {
+    if (!first)
```

Comment:
I would have preferred
````
if (first)
  first = false;
else
  ss << sep;
````
but mebbe that's too nitpicky.

Reply:
Reminds me of https://github.com/monero-project/monero/blob/5fee85616d408dd2e30c3aa104c4c24566a6f480/src/blockchain_utilities/blockchain_export.cpp#L47 (which I removed in a cleanup patch) seems like there should be something in tools:: to just do this in general.

Reply:
I'd actually first put it in src/common/util but removed it  because it'd be a template, and I'd have to instanciate for vector/etc in it or put the source in the header :P 
About the extra else, I don't really mind. There's an extra branch in that version, but maybe the compiler will work it out ?